tests/compiletest.rs

```diff
@@ -61,6 +63,16 @@ fn run_tests(mode: Mode, path: &str, target: Option<String>) -> Result<()> {
         path_filter: path_filter.collect(),
         program: miri_path(),
         output_conflict_handling,
+        manifest_path: use_std
+            .then(|| std::env::current_dir().unwrap().join("test_dependencies").join("Cargo.toml")),
+        dependency_builder: Some(DependencyBuilder {
+            program: PathBuf::from("bash"),
+            args: vec![
+                std::env::current_dir()?.join("miri").display().to_string(),
```

Comment:
Can you change the type of `args` such that it contains `OsString` rather than `String`?